### PR TITLE
Software use case rewrite

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

**Comment by rjb4standards** [comment]
@yogeshbdeshpande the revisions seem to place greater emphasis on package managers, removing the original intent. The original intent was much broader and could be applied to a wide swath of software distribution, i.e. GitHub, app stores, and package manager repos.

**Comment by yogeshbdeshpande** [comment]
> @yogeshbdeshpande the revisions seem to place greater emphasis on package managers, removing the original intent. The original intent was much broader and could be applied to a wide swath of software distribution, i.e. GitHub, app stores, and package manager repos.

Sure, we can broaden the scope. Not a problem!

**Review by kaywilliams — COMMENTED** [review]
Previously I had suggested focusing on lower-level use cases, but after reviewing the document with Yogesh and Henk, I believe the current approach works.

**Comment by yogeshbdeshpande** [comment]
> should be on satisfying lower-level use case

We had a detail discussion about this between me and Henk. Basically, I was all for the support of going to depict a real world example for each of the situation. However, this makes the document excessively long and verbose. 

We should make a judgment as a team as to:

1. Where should the Use Case document reside ? Some members feel within Arch. Document, some feel as a separate RFC.
2. Given the number of scenarios, should we abstract it to generic use case or we depict a specific example, next to each of them ?
3. Is too long a document (when a separate RFC) acceptable to the group ?

File: draft-birkholz-scitt-software-use-cases.md [hunk 1]
```diff
@@ -85,55 +85,53 @@ deployment chain diagram here
...
 
-* to understand and verify that an actual trust bond exists between the Supplier of a certain software component package and the Signing Authority of that software component package.
+* to understand and verify that an actual relationship link exists between the Supplier of a certain software component package and the Signing Authority of that software component package.
 
 There is no standardized way to:
 
-* enable the consumer to verify that a trust bond for a certain software component package exists and is still valid.
+* enable the consumer to verify that a relationship link for a certain software component package  between the supplier and the distributor(its signing authority) exists and is still valid.
+
+## Multi party Evaluation of a Released Software Component
```
Comment (yogeshbdeshpande) [inline draft-birkholz-scitt-software-use-cases.md]:
```suggestion
## Multi Stakeholder Evaluation of a Released Software Component
```

Comment (rjb4standards) [reply draft-birkholz-scitt-software-use-cases.md]: There is no requirement that the distributor be the signing authority. The signing authority is whomever the software supplier chooses, and many cases the supplier and signing authority are the same entity. That's how it is with our SAG-PM software; REA is both supplier and signing authority.

Comment (yogeshbdeshpande) [reply draft-birkholz-scitt-software-use-cases.md]: @rjb4standards are you available sometime tomorrow to have a discussion with me and Henk?
Also what is your email address? We want to schedule a meeting with you..

Comment (rjb4standards) [reply draft-birkholz-scitt-software-use-cases.md]: dick@reliableenergyanalytics.com   I can meet tomorrow

**Comment by yogeshbdeshpande** [comment]
@henkbirkholz @rjb4standards @kaywilliams I have redone the PR post our meeting today. 

Please have a look..

**Comment by yogeshbdeshpande** [comment]
> > @yogeshbdeshpande the revisions seem to place greater emphasis on package managers, removing the original intent. The original intent was much broader and could be applied to a wide swath of software distribution, i.e. GitHub, app stores, and package manager repos.
> 
> Sure, we can broaden the scope. Not a problem!

This has been resolved now!

File: draft-birkholz-scitt-software-use-cases.md [hunk 2]
```diff
@@ -85,57 +85,51 @@ deployment chain diagram here
 
 # Software Supply Chain Use Cases
 
-## Trustworthiness Relationship between Product Supplier and the Distributor of the Product
+## Verifiable Authenticity in Software Distribution Systems
```
Comment (OR13) [inline draft-birkholz-scitt-software-use-cases.md]: I like the section header change

Comment (kaywilliams) [reply draft-birkholz-scitt-software-use-cases.md]: +1

File: draft-birkholz-scitt-software-use-cases.md [hunk 3]
```diff
@@ -244,35 +232,22 @@ Even with a robust firmware update system, the following problems remain as give
...
-
-* However, there are no "Verifiable Proofs" of the statement made by Vendor OS-X.
-
-* MCW, Smart Cars Ltd., and Customer-Y now have to trust without the ability to verify the claims made by Vendor OS-X.
-
-* Vendor OS-X thinks there is some mistake on the part of MCW that has led to this situation.
+* ISV attempts to show that it did all the steps correctly. It does disclose information about the binary they delivered. In addition, they also reveal their build environment and the architecture they used during the build.
 
-* The deadlock continues, with no clear resolution.
+* However, there are no "Verifiable Proofs" of the statement made by ISV. All the stakeholders, in the ecosystem (end user, car manufacturer and the integrator) has to trust without any ability to verify the claims made by the ISV.
```
Comment (OR13) [inline draft-birkholz-scitt-software-use-cases.md:250]: I like the change.

**Review by OR13 — APPROVED** [review]
great improvements

**Review by kaywilliams — APPROVED** [review]

**Pull request merged**